util/test/demos/CMakeLists.txt

```diff
@@ -101,13 +98,28 @@ set(OPENGL_SRC
         gl/gl_vertex_attr_zoo.cpp)
 
 set(SRC main.cpp
+        3rdparty/fmt/format.cc
         test_common.cpp
-        texture_zoo.cpp
-        linux/linux_platform.cpp
-        linux/linux_window.cpp)
+        texture_zoo.cpp)
 
 project(demos)
-add_executable(demos ${SRC} ${VULKAN_SRC} ${OPENGL_SRC})
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
Was not sure what is the preferred style here. I copied the setup from the main CMakeLists.txt configuration file.

Could merge the "-Werror" directly into line 108

Reply:
I think this is fine as-is.